nncf/quantization/algorithms/bias_correction/algorithm.py

```diff
@@ -536,12 +560,15 @@ def get_statistic_points(self, model: TModel, graph: NNCFGraph) -> StatisticPoin
 
         # Then we need also to collect model input statistics to prevent cases when nodes with bias have no input data.
         for input_node in model_inputs:
-            # We assume that input node has only one output port
-            input_name = input_node.node_name
-            if input_name in statistic_container:
+            # We need to map all input node outputs as nodes with statistic
+            for edge in graph.get_output_edges(input_node):
+                input_id = (edge.to_node.node_name, edge.input_port_id)
+                output_id = (input_node.node_name, edge.output_port_id)
+                self._collected_stat_inputs_map[input_id] = output_id
```

Comment:
Can we use a `dataclass` for `input_id`/`output_id` to improve readability?

Reply:
How do you imagine that? `input_id/output_id` are simple tuple of string and int values.

Reply:
It is unclear for me what are `input_id`/`output_id` stands for, could you specify what is it first?

Reply:
Please, look at the 565-566 lines.

Reply:
Could you please specify somewhere that this pair stands for a node name + port id for an input/output? In a docstring or in a comment?

Reply:
Is your suggestion above
> List of the input IDs: pairs of node names and correspondent input port ids. Each pair denotes the sub-graph beginning.

not enough?